.github/workflows/dry-run.yml

```diff
@@ -0,0 +1,38 @@
+name: Dry Run
+on:
+  push:
+  # Run on all branches in our repo, so that we test dry-run for our PRs too
+  # branches: [main]
+  # Don't run on forks, which don't have access to secrets
+  # pull_request:
+  #  branches: [main]
```

Comment:
is this part still wip?

Reply:
It isn't, we can remove the lines that are commented-out.

Reply:
I removed the commented-out code in [a1472bd](https://github.com/filecoin-station/spark-evaluate/pull/204/commits/a1472bd301032f3995ce1e844f34b5e214658d57).

LGTY now?